components/staging/staging-hearings-reform.tf

```diff
@@ -0,0 +1,15 @@
+#data "local_file" "staging-hearings-reform-configuration" {
```

Comment:
please don't leave commented out code either include or leave out, shouldn't be any harm adding this now

Reply:
Removed the comments for staging